src/Draco.LanguageServer/Translator.cs

```diff
@@ -62,4 +64,72 @@ public static LspModels.Position ToLsp(CompilerApi.Syntax.SyntaxPosition positio
         Line = (uint)position.Line,
         Character = (uint)position.Column,
     };
+
+    public static LspModels.CompletionItem ToLsp(CompilerApi.CodeCompletion.CompletionItem item)
+    {
+        var result = new LspModels.CompletionItem()
+        {
+            // TODO: Maybe we will have completions that don't just append text or that have multiple edits
+            Label = item.DisplayText,
+            Kind = ToLsp(item.Kind),
+        };
+        if (item.Edits.Length > 0) result.TextEdit = ToLsp(item.Edits[0]);
+        var aditionalEdits = new List<LspModels.TextEdit>();
+        for (int i = 1; i < item.Edits.Length; i++)
+        {
+            aditionalEdits.Add(ToLsp(item.Edits[i]));
+        }
+        result.AdditionalTextEdits = aditionalEdits;
+        if (item.Symbols.FirstOrDefault() is CompilerApi.Semantics.ITypedSymbol typed)
+        {
+            result.Detail = item.Symbols.Count() == 1 ? typed.Type.ToString() : $"{item.Symbols.Count()} overloads";
+        }
+        if (!string.IsNullOrEmpty(item.Symbols.FirstOrDefault()?.Documentation)) result.Documentation = new LspModels.MarkupContent()
+        {
+            Kind = LspModels.MarkupKind.Markdown,
+            Value = item.Symbols.First().Documentation,
+        };
+        return result;
```

Comment:
Please refactor to contain less mutations. Accumulate edits, markup content, ...
Then finally construct a completion item